addon_service/abstract/configured_addon/models.py

```diff
@@ -40,16 +40,14 @@ class ConfiguredAddon(AddonsServiceBaseModel):
         base_account: AuthorizedAccount
         authorized_resource: ResourceReference
 
-    class Meta:
-        abstract = True
-
     @property
     def display_name(self):
         return self._display_name or self.base_account.display_name
 
     @display_name.setter
     def display_name(self, value: str):
-        self._display_name = value
+        if value:
+            self._display_name = value
```

Comment:
This is because frontend passes empty `display_name` in request and this causes many issues. Furthermore, are there any cases, when we'd need to clear `display_name`?

Reply:
hm, that seems like frontend misbehavior... but failing fixing that, conforming to frontend misbehavior seems like a concern for the serializer more than the model -- my concern is not supporting that use case, but hoping for consistent/unsurprising behavior in code

but this is a small case that doesn't matter much, i was only curious why